### ISSUE TITLE
[Amélioration] Téléchargement des dataviz

### ISSUE THREAD

**Issue by pbarille** [spelhate/mreport#183]
Ajout d'un bouton dans les rapports permettant de télécharger la dataviz au format image.

**Comment by lecault** [comment]
Il n'était pas question aussi de télécharger les données ?

**Comment by pbarille** [comment]
Tout à fait, c'est l'issue https://github.com/geobretagne/mreport/issues/178
